specs/PrintToPdf.md

```diff
@@ -0,0 +1,303 @@
+                &printSettings));
+        }
+
+        if (printSettings)
+        {
+            if (!useDefaultOrientation)
+            {
+                CHECK_FAILURE(printSettings->put_Orientation(
+                    COREWEBVIEW2_PRINT_ORIENTATION_LANDSCAPE));
+            }
+            CHECK_FAILURE(m_webView4->PrintToPdf(
+                openFileName.lpstrFile, printSettings.get(),
+                Callback<ICoreWebView2PrintToPdfCompletedHandler>(
+                    [](HRESULT errorCode, LPCWSTR resultFilePath) -> HRESULT {
+                        CHECK_FAILURE(errorCode);
+                        std::wstringstream formattedMessage;
+                        formattedMessage << "ResultFilePath: " << resultFilePath;
+                        MessageBox(nullptr, formattedMessage.str().c_str(),
+                            L"Print to PDF Completed", MB_OK);
+                        return S_OK;
+                    })
+                    .Get()));
+        }
+    }
+}
+```
+
+```c#
+// Prints current page to PDF using the default path and settings. If requested,
+// the orientation is changed to landscape.
+async void PrintToPdfCmdExecuted(object target, ExecutedRoutedEventArgs e)
+{
+    CoreWebView2PrintSettings printSettings =
+        WebViewEnvironment.CreateDefaultPrintSettings();
+
+    string orientationString = e.Parameter.ToString();
+    if (printSettings != null && orientationString == "Landscape")
+    {
+        printSettings.Orientation =
+            CoreWebView2PrintOrientation.Landscape;
+    }
+    string resultFilePath = await webView.CoreWebView2.PrintToPdfAsync(
+        "" /* use default path*/, printSettings);
+    MessageBox.Show(this, resultFilePath, "Print To PDF Completed");
+}
+```
+# Remarks
+
+# API Notes
+
+
+# API Details
+```
+interface ICoreWebView2_4;
+interface ICoreWebView2Environment5;
+interface ICoreWebView2PrintSettings;
+interface ICoreWebView2PrintToPdfCompletedHandler;
+
+
+/// The orientation for printing, used by the `Orientation` property on
+/// `ICoreWebView2SPrintSettings`. Currently only printing to PDF
+/// is supported.
+[v1_enum]
+typedef enum COREWEBVIEW2_PRINT_ORIENTATION
+{
+  /// Print the page(s) in portrait orientation.
+  COREWEBVIEW2_PRINT_ORIENTATION_PORTRAIT,
+
+  /// Print the page(s) in landscape orientation.
+  COREWEBVIEW2_PRINT_ORIENTATION_LANDSCAPE,
+} COREWEBVIEW2_PRINT_ORIENTATION;
+
+[uuid(9eae81c3-fe02-4084-b475-903a4ed9252e), object, pointer_default(unique)]
+interface ICoreWebView2_4 : IUnknown {
+  /// Print the current page to PDF asynchronously with the provided settings.
+  /// See `ICoreWebView2PrintSettings` for description of settings.
+  ///
+  /// Use `resultFilePath` to specify the path to the PDF file. The host should
+  /// provide an absolute path, including file name. If the path
+  /// points to an existing file, the file will be overwritten. If
+  /// `resultFilePath` is invalid, `isSuccessful` in
```

Comment:
There is no isSuccessful - you must mean the async completion HRESULT error?

Reply:
Yes sorry I hadn't updated the sample code portion. It's up to date now